<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>543</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>543</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>61</y>
      <width>73</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Source dir:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRecursive">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Recursive</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLabel" name="SourceDirLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Log:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCmakeGeneration">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate CMAKE configuration</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditLogs">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>441</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ChooseFileButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
